module/entityAssessors/helper.js

```diff
@@ -214,15 +227,17 @@ module.exports = class entityAssessorHelper {
                 let creatorId = userId;
 
                 assessorData = await Promise.all(assessorData.map(async (assessor) => {
+                    assessor["userId"] = userIdByExternalId[assessor.externalId];
+                    if(assessor.parentId) assessor["parentId"] = userIdByExternalId[assessor.parentId];
                     let assessorEntityArray = new Array
-                    assessor.entities.split(",").forEach(assessorSchool => {
-                        if (entityData[assessorSchool.trim()])
-                            assessorEntityArray.push(entityData[assessorSchool.trim()])
+                    assessor.entities.split(",").forEach(assessorEntity => {
```

Comment:
Why not use the same code already done above ?

@ajithtunerlabs

Reply:
here we need to use entityData in 163 no need of that.